src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -299,13 +333,66 @@ function generateFrontendPerformanceEventView(
   return EventView.fromNewQueryWithLocation(savedQuery, location);
 }
 
-export function generatePerformanceEventView(organization, location) {
+function generateFrontendNavigationPerformanceEventView(
+  organization: LightWeightOrganization,
+  location: Location
+): EventView {
+  const {query} = location;
+
+  const hasStartAndEnd = query.start && query.end;
+  const savedQuery: NewQuery = {
+    id: undefined,
+    name: t('Performance'),
+    query: 'event.type:transaction',
+    projects: [],
+    fields: [
+      'key_transaction',
+      'transaction',
+      'project',
+      'tpm()',
+      'p50(transaction.duration)',
+      'p75(transaction.duration)',
+      'p95(transaction.duration)',
+      'count_unique(user)',
+      `user_misery(${organization.apdexThreshold})`,
+    ],
+    version: 2,
+  };
+
+  if (!query.statsPeriod && !hasStartAndEnd) {
+    savedQuery.range = DEFAULT_STATS_PERIOD;
+  }
+  savedQuery.orderby = decodeScalar(query.sort) || '-tpm';
+
+  const searchQuery = decodeScalar(query.query) || '';
+  const conditions = tokenizeSearch(searchQuery);
+  conditions.setTagValues('event.type', ['transaction']);
```

Comment:
Hm... do we need to add the `<15m` filter in navigation if we're using duration?

Reply:
Yeah we do.